egs/bentham/steps/train_lstm1d.sh

```diff
@@ -60,6 +60,9 @@ Options:
 source "$(pwd)/utils/parse_options.inc.sh" || exit 1;
 [ $# -ne 0 ] && echo "$help_message" >&2 && exit 1;
 
+export CUDA_VISIBLE_DEVICES=$((gpu-1))
+gpu=1
```

Comment:
Don't override the user's choice. The same applies to the rest of scripts.

Reply:
The -1 is because the environment variable's GPU numbering is from zero and not one like Lua. Once the GPU has been selected from the environment variable then that is the only GPU available, hence from Lua's perspective it is GPU number 1.

Here's an example to make it clearer. Previously I would start the training on the second GPU with:
CUDA_VISIBLE_DEVICES=1 laia-train-ctc --gpu 1 ...
and on the third GPU with:
CUDA_VISIBLE_DEIVCES=2 laia-train-ctc --gpu 1 ...

Hope that clarifies what is going on, or did I misunderstand what you meant?

Reply:
Thanks for the clarification, now I understand it.